README.md

```diff
@@ -7,6 +7,8 @@ Arrowp-qq
 A preprocessor for arrow notation 
 based on the arrowp preprocessor developed by Ross Paterson <ross@soi.city.ac.uk>.
 
+Forked from Jose Iborra's arrowp-qq preprocessor, updating package version constraints and fixing resulting code breakage. It should now build under Cabal v3.
```

Comment:
is this still relevant?
```suggestion
```

Reply:
I'd remove it also; was moreso a disclaimer while it was still an unapproved fork.